acceptance-tests/src/main/java/io/blueocean/ath/AthModule.java

```diff
@@ -4,20 +4,15 @@
 import com.google.inject.assistedinject.FactoryModuleBuilder;
 import com.google.inject.name.Names;
 import com.offbytwo.jenkins.JenkinsServer;
-import io.blueocean.ath.factory.ActivityPageFactory;
-import io.blueocean.ath.factory.BranchPageFactory;
-import io.blueocean.ath.factory.FreestyleJobFactory;
-import io.blueocean.ath.factory.MultiBranchPipelineFactory;
-import io.blueocean.ath.factory.ClassicPipelineFactory;
-import io.blueocean.ath.factory.RunDetailsArtifactsPageFactory;
-import io.blueocean.ath.factory.RunDetailsPipelinePageFactory;
+import io.blueocean.ath.factory.*;
```

Comment:
I seem to recall that using wildcard imports violates some style rules in Jenkins projects, I think @vivek pointed that out to me once. If you're using IDEA and search for "class count to use import with" in settings, you'll find an option. I set it to 100.